### ISSUE TITLE
wrap climlab state variables and axes in xarray data structures

### ISSUE THREAD

**Issue by rabernat** [brian-rose/climlab#50]
By wrapping the model state variables in xarray data structures, it would greatly simplify plotting and general user interaction. It would also enable easy serialization of model states to netCDF files. Not sure how much work it would be, but it would be very cool!

**Comment by brian-rose** [comment]
I completely agree, and it's on my to-do list. You should be able to take a model object and do something like
```
mymodel.to_xarray()
```
which would return a `xarray.Dataset` object of state variables and diagnostics.

I actually don't think it would be very difficult to implement because it need not touch the climlab internals at all.

**Comment by brian-rose** [comment]
As of https://github.com/brian-rose/climlab/commit/547b510c6adf61e7a049bc3d0264f191704fcbee (release v0.6.1), partial xarray integration is here!

There are now several `to_xarray()` methods available to wrap climlab model variables in `xarray.DataArray` and `xarray.Dataset` objects.

[See the release note](https://github.com/brian-rose/climlab/releases/tag/v0.6.1)

**Comment by brian-rose** [comment]
Keeping this issue open because deeper xarray integration would be better. 

I am hoping to use `xarray.DataArray` as the native format for climlab internals in the future, and get rid of the `climlab.Field` object entirely.
